openhexa/toolbox/iaso/iaso.py

```diff
@@ -0,0 +1,34 @@
+import io
+
+from openhexa.toolbox.iaso.api_client import ApiClient
+import polars as pl
+
+
+class IASO:
+    def __init__(self, client: ApiClient) -> None:
+        self.api_client = client
+
+    def get_projects(self, **kwargs) -> dict:
+        response = self.api_client.request("GET", "/api/projects", params=kwargs)
+        return response.json().get("projects")
+
+    def get_org_units(self, **kwargs) -> dict:
+        response = self.api_client.request("GET", "/api/orgunits", params=kwargs)
+        return response.json().get("orgUnits")
+
+    def get_all_submissions_forms(self) -> dict:
+        response = self.api_client.request("GET", "/api/forms")
+        forms = response.json().get("forms")
+        return forms
+
+    def get_submissions_forms(self, org_units=None, projects=None, **kwargs) -> dict:
```

Comment:
We have to have a pagination mechanism. What happens here if the user calls this ? Do we return everything or is it hidden that he's just going to get a page of x? elements ?

Reply:
Indeed just remembered I forgot the default pagination, it can be added in kwargs. Will add a default one